crates/ra_project_model/src/cargo_workspace.rs

```diff
@@ -83,6 +83,7 @@ pub struct PackageData {
     pub dependencies: Vec<PackageDependency>,
     pub edition: Edition,
     pub features: Vec<String>,
+    pub cfgs: Vec<PathBuf>,
```

Comment:
TIL [`BuildScript::cfgs`][1] in cargo_meta is `Vec<PathBuf>` !

But why ? 

[1]: https://github.com/oli-obk/cargo_metadata/blob/da4d25cf0e6d4a5a1d787eefc326840cc34e1a72/src/messages.rs#L75

Reply:
Might well be a copy-paste error in `cargo_metadata`; the cargo documentation [says](https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-cfg):

> single identifier, or any arbitrary key/value pair [...] key should be a Rust identifier, the value should be a string

which should well be covered by `String` rather than `PathBuf`/`OsString`. Should I send an issue/PR to `cargo_metadata`?

Reply:
Sound reasonable !

Reply:
oli-obk/cargo_metadata#112 has been merged and I changed the code to minimize the needed changes once that hits `crates.io`.